verification/sign.go

```diff
@@ -0,0 +1,189 @@
+// Licensed under the Apache-2.0 license
+
+package verification
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/x509"
+	"errors"
+	"math/big"
+	"testing"
+)
+
+// Obtain and validate signature of asymmetric signing.
+func TestAsymmetricSigning(d TestDPEInstance, c DPEClient, t *testing.T) {
+	useSimulation := false
+	handle := getInitialContextHandle(d, c, t, useSimulation)
+	// Get digest size
+	profile, err := GetTransportProfile(d)
+	if err != nil {
+		t.Fatalf("Could not get profile: %v", err)
+	}
+
+	digestLen := profile.GetDigestSize()
+	if digestLen != 32 && digestLen != 48 {
+		t.Fatalf("[FATAL]: Unsupported digest size.")
+	}
```

Comment:
nit: probably not necessary. The client will already fail if the lengths are wrong.

Reply:
ok removed